### PR TITLE
Fixed context outputs - FailedInstances Script

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/27388

## Description
Fixed context outputs.
Now we have:
 * FailedInstances
 * SuccessInstances
 * InstancesCount

## Screenshots
image

## Minimum version of Demisto
- [x] 4.0.0

## Does it break backward compatibility?
   - [x] Yes
       - Further details: Changed a bit the outputs paths. The script was not released yet in the content release.
   - [ ] No

### CODE REVIEW

**Comment by Shellyber** [comment]
@altmannyarden FYI

**Comment by Shellyber** [comment]
@kirbles19 Please review the YML file, release notes and README files.

File: Packs/CommonScripts/ReleaseNotes/1_2_32.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### FailedInstances
+- Separated the context paths of the instances to *FailedInstances*, *InstancesCount* and *SuccessInstances*.
```
Comment (Itay4) [inline Packs/CommonScripts/ReleaseNotes/1_2_32.md:4]: please mention here this introduces breaking change for 6.0 and above

File: Packs/CommonScripts/Scripts/script-FailedInstances.yml [hunk 2]
```diff
@@ -93,14 +95,29 @@ outputs:
 - contextPath: FailedInstances.status
   description: Status of the instance.
   type: string
-- contextPath: FailedInstances.failureCount
-  description: The number of failed instnaces.
-  type: string
-- contextPath: FailedInstances.successCount
+- contextPath: InstancesCount.FailedCount
+  description: The number of failed instances.
+  type: number
```
Comment (Itay4) [inline Packs/CommonScripts/Scripts/script-FailedInstances.yml:100]:
```suggestion
  type: string
```

File: Packs/CommonScripts/Scripts/script-FailedInstances.yml [hunk 3]
```diff
@@ -93,14 +95,29 @@ outputs:
...
+- contextPath: InstancesCount.SuccessCount
   description: The number of working instances.
-  type: string
-- contextPath: FailedInstances.totalCount
+  type: number
```
Comment (Itay4) [inline Packs/CommonScripts/Scripts/script-FailedInstances.yml:103]:
```suggestion
  type: string
```

File: Packs/CommonScripts/Scripts/script-FailedInstances.yml [hunk 4]
```diff
@@ -93,14 +95,29 @@ outputs:
...
+- contextPath: InstancesCount.TotalCount
   description: The number of total enabled instances.
+  type: number
```
Comment (Itay4) [inline Packs/CommonScripts/Scripts/script-FailedInstances.yml:106]:
```suggestion
  type: string
```

**Pull request merged**